### PR TITLE
Makefile command to publish scala jars to nexus and s3

### PR DESCRIPTION
**Pull request by itaiad200**

### CODE REVIEW

File: clients/spark/build.sbt [hunk 1]
```diff
@@ -84,6 +84,7 @@ lazy val assemblySettings = Seq(
 
 // Don't publish root project
 root / publish / skip := true
+root / publishAll := false
```
Comment (johnnyaug) [inline clients/spark/build.sbt:87]: works even without this

File: Makefile [hunk 2]
```diff
@@ -200,6 +202,10 @@ proto: ## Build proto (Protocol Buffers) files
 	$(PROTOC) --proto_path=pkg/graveler/committed --go_out=pkg/graveler/committed --go_opt=paths=source_relative committed.proto
 	$(PROTOC) --proto_path=pkg/graveler --go_out=pkg/graveler --go_opt=paths=source_relative graveler.proto
 
+publish-scala: ## sbt publish jars to nexus and s3 bucket
+	cd clients/spark && sbt publish
```
Comment (johnnyaug) [inline Makefile:206]:
```suggestion
	cd clients/spark && sbt publishAll
```
Or am I missing something?

**Review by johnnyaug — CHANGES_REQUESTED** [review]
Clean!

File: Makefile (hunk 2 above)
Comment (itaiad200) [reply Makefile]: Good catch!

**Review by johnnyaug — APPROVED** [review]

**Pull request merged**